modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -983,11 +997,20 @@ void ONNXImporter::populateNet(Net dstNet)
             {
                 Mat inp0 = getBlob(node_proto, constBlobs, 0);
                 Mat inp1 = getBlob(node_proto, constBlobs, 1);
-                if (inp0.size != inp1.size)
+                if (inp0.size != inp1.size && inp1.total() != 1)
                     CV_Error(Error::StsNotImplemented, "Constant multiply with different shapes");
 
                 Mat out;
-                if (isDiv)
+                if (inp1.total() == 1)
+                {
+                    if (inp1.type() == CV_32F)
+                        out = (isDiv ? 1.0 / inp1.at<float>(0) : inp1.at<float>(0)) * inp0;
+                    else if (inp1.type() == CV_32S)
+                        out = (isDiv ? 1.0 / inp1.at<int>(0) : inp1.at<int>(0)) * inp0;
```

Comment:
What's the problem with single element matrices?

Reply:
Problem with divide or multiply for single element matrix and non-single element matrix

Reply:
`inp0` or `inp1`? Can you show an example with the shapes?

Reply:
```
inp0.size = [N, 1]
inp1.size = [1]
```

Reply:
Will
```cpp
out =  inp0 / inp1
```
here?